crates/ln-dlc-node/src/node.rs

```diff
@@ -564,6 +565,7 @@ impl Node {
             .payment_hash(sha256::Hash::from_slice(&payment_hash.0)?)
             .payment_secret(payment_secret)
             .timestamp(SystemTime::now())
+            .min_final_cltv_expiry(133) // default: 9 todo: check why lnd payments only work with a higher number
```

Comment:
I don't understand the beginning of the comment

Reply:
I've updated the comment. I hope it's clearer now.